OSDOCS#6797: linking Operator update compatbility app in OTA docs (#70911)
openshift/openshift-docs: updating/updating_a_cluster/updating-cluster-cli.adoc

* Have access to the cluster as a user with `admin` privileges.
See xref:../../authentication/using-rbac.adoc#using-rbac[Using RBAC to define and apply permissions].
* Have a recent xref:../../backup_and_restore/control_plane_backup_and_restore/backing-up-etcd.adoc#backup-etcd[etcd backup] in case your update fails and you must restore your cluster to a previous state.
* Have a recent xref:../../backup_and_restore/application_backup_and_restore/installing/oadp-backup-restore-csi-snapshots.adoc[Container Storage Interface (CSI) volume snapshot] in case you need to restore persistent volumes due to a pod failure.

> * Support for {op-system-base}7 workers is removed in {product-title} {product-version}. You must replace {op-system-base}7 workers with {op-system-base}8 or {op-system} workers before updating to {product-title} {product-version}. Red{nbsp}Hat does not support in-place {op-system-base}7 to {op-system-base}8 updates for {op-system-base} workers; those hosts must be replaced with a clean operating system install.

Author: I'm not sure the "support has been removed" statement is necessary any longer, if it's okay with you I'd like to change this to the same wording as we decided for the [web console module](https://github.com/openshift/openshift-docs/pull/70911/files#diff-18e434b00c7ff90de0321a1ed4af6f5434c5fe90182bd58bd733c65a744c5355):
```suggestion
* Your {op-system-base}7 workers are replaced with {op-system-base}8 or {op-system} workers. Red{nbsp}Hat does not support in-place {op-system-base}7 to {op-system-base}8 updates for {op-system-base} workers; those hosts must be replaced with a clean operating system install.
```

Let me know what you think!

> * Support for {op-system-base}7 workers is removed in {product-title} {product-version}. You must replace {op-system-base}7 workers with {op-system-base}8 or {op-system} workers before updating to {product-title} {product-version}. Red{nbsp}Hat does not support in-place {op-system-base}7 to {op-system-base}8 updates for {op-system-base} workers; those hosts must be replaced with a clean operating system install.

Reviewer: I see the preview page is updated, it looks fine to me.
```
Your RHEL7 workers are replaced with RHEL8 or RHCOS workers. Red Hat does not support in-place RHEL7 to RHEL8 updates for RHEL workers; those hosts must be replaced with a clean operating system install.
```

[IMPORTANT]
====

> * When an update is failing to complete, the Cluster Version Operator (CVO) reports the status of any blocking components while attempting to reconcile the update. Rolling your cluster back to a previous version is not supported.  If your update is failing to complete, contact Red{nbsp}Hat support.

Author: Thanks for catching this!
